tox.ini

```diff
@@ -20,7 +20,7 @@ commands =
 setenv = 
     DJANGO_SETTINGS_MODULE = test_settings
     PYTHONPATH = {toxinidir}
-whitelist_externals = 
+allowlist_externals = 
```

Comment:
`whitelist_externals` was deprecated and has been replaced with `allowlist_externals`